bee-network/src/peer/error.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use libp2p::{Multiaddr, PeerId};
+
+/// [`PeerList`] errors.
+#[derive(Clone, Debug, thiserror::Error)]
+pub enum Error {
+    /// A failure due to an address being added twice.
+    #[error("Already added that address: {}", .0)]
+    AddressIsDuplicate(Multiaddr),
+
+    /// A failure due to an address being banned.
+    #[error("Already banned that address: {}", .0)]
+    AddressIsBanned(Multiaddr),
+
+    /// A failure due to an address being one of the bind addresses.
+    #[error("Address is one of the local bind addresses: {}", .0)]
+    AddressIsLocal(Multiaddr),
+
+    /// A failure due to an address being already unbanned.
+    #[error("Already unbanned that address: {}", .0)]
+    AddressIsUnbanned(Multiaddr),
+
+    /// A failure due to a peer id being equal to the local id.
+    #[error("Peer matches the local Id: {}", .0)]
+    PeerIsLocal(PeerId),
+
+    /// A failure due to a peer not being present in the peerlist.
+    #[error("Not present peer: {}", .0)]
+    PeerNotPresent(PeerId),
+
+    /// A failure due to attempting to add a peer twice.
+    #[error("Already added that peer: {}", .0)]
+    PeerIsDuplicate(PeerId),
+
+    /// A failure due to a peer id being in the peer-id banlist.
```

Comment:
Sometimes you write `peer id` and sometimes `peer-id`

Reply:
don't be more german than I am. One german is already enough :trollface: